packages/@magic-sdk/provider/src/modules/auth.ts

```diff
@@ -81,29 +83,26 @@ export class AuthModule extends BaseModule {
    * of 15 minutes)
    */
   public loginWithEmailOTP(configuration: LoginWithEmailOTPConfiguration) {
-    createDeprecationWarning({
-      method: 'auth.loginWithEmailOTP()',
-      removalVersions: ProductConsolidationMethodRemovalVersions,
-      useInstead: '@magic-ext/auth auth.loginWithEmailOTP()',
-    }).log();
-    const { email, showUI } = configuration;
+    const { email, showUI, deviceCheckUI } = configuration;
     const requestPayload = createJsonRpcRequestPayload(
       this.sdk.testMode ? MagicPayloadMethod.LoginWithEmailOTPTestMode : MagicPayloadMethod.LoginWithEmailOTP,
-      [{ email, showUI }],
+      [{ email, showUI, deviceCheckUI }],
```

Comment:
@Ethella - Given [our history](https://www.notion.so/magiclabs/Email-OTP-Modal-was-Hidden-on-Native-Mobile-SDK-s-Due-To-Magic-SDK-Update-301ccd9a74bb4de9abe63697ce4ba149?pvs=4) with this bit of code can you confirm this won't affect our native SDKs currently implementations of `LoginWithEmailOTP`?